docs/sdks/react-native-sdk/integrate-with-beyondidentity.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Integrate With Beyond Identity
+sidebar_position: 1
+---
+
+This guide describes how to use Beyond Identity for authentication during an OAuth2 authorization flow.
+
+## Prerequisites
+
+- [Using Beyond Identity for Authentication](../../using-bi-for-auth)
+- [React Native SDK Overview](overview)
+
+Before calling [`Embedded.authenticate`](overview#authentication), we must [Authorize With Beyond Identity](integrate-with-beyondidentity#authorize-with-beyond-identity)
+
+## Authorize With Beyond Identity
+
+### Using a WebView
```

Comment:
nit: I changed the language from "Using a WebView" to "Using the Web" since RFC 8252 does not advise WebViews

Reply:
Hm but in this case, it is still using a webview.